### ISSUE TITLE
Column width wrapping

### ISSUE THREAD

**Issue by jlixfeld** [respawner/peering-manager#63]
### Issue type
[ ] Feature request 
[X] Bug report      
[ ] Documentation   

### Environment
* Python version:   3.5.3
* Peering Manager version:   0.99-dev

### Description

Column widths are too narrow for some text and wrap without taking available screen resolution into consideration.

IE:
<img width="3339" alt="screen shot 2018-11-12 at 7 13 25 am" src="https://user-images.githubusercontent.com/17339017/48347924-bc404900-e64d-11e8-9954-8d0b7bef19a8.png">

**Comment by respawner** [comment]
Would you mind testing with ccca1c8? What screen resolution do you use?

**Comment by jlixfeld** [comment]
Issues after updating.

```
There was a problem with your request. Please contact an administrator.

The complete exception is provided below:

<class 'django.db.utils.OperationalError'>
no such table: peering_routingpolicy
```

```
root@peeringmanager:/opt/peering-manager# ./scripts/upgrade.sh
Cleaning up old Python bytecode (find . -name "*.pyc" -delete)...
Updating required Python dependencies (pip3 install -r requirements.txt --upgrade)...
Requirement already up-to-date: Django==2.1.3 in /usr/local/lib/python3.5/dist-packages (from -r requirements.txt (line 1))
Requirement already up-to-date: django-filter==2.0.0 in /usr/local/lib/python3.5/dist-packages (from -r requirements.txt (line 2))
Collecting django-tables2==2.0.3 (from -r requirements.txt (line 3))
  Downloading https://files.pythonhosted.org/packages/06/c0/1bb8597df1cc72f73a47f9b46c51a6d81c80f9f2defc894cb00166f29d56/django_tables2-2.0.3-py2.py3-none-any.whl (89kB)
    100% |████████████████████████████████| 92kB 3.6MB/s
Requirement already up-to-date: Jinja2==2.10 in /usr/local/lib/python3.5/dist-packages (from -r requirements.txt (line 4))
Requirement already up-to-date: napalm==2.3.3 in /usr/local/lib/python3.5/dist-packages (from -r requirements.txt (line 5))
Requirement already up-to-date: MarkupSafe==1.1.0 in /usr/local/lib/python3.5/dist-packages (from -r requirements.txt (line 6))
Requirement already up-to-date: py-gfm==0.1.4 in /usr/local/lib/python3.5/dist-packages (from -r requirements.txt (line 7))
Requirement already up-to-date: pytz==2018.7 in /usr/local/lib/python3.5/dist-packages (from -r requirements.txt (line 8))
Requirement already up-to-date: requests==2.20.1 in /usr/local/lib/python3.5/dist-packages (from -r requirements.txt (line 9))
Requirement already up-to-date: future in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Collecting scp (from napalm==2.3.3->-r requirements.txt (line 5))
  Downloading https://files.pythonhosted.org/packages/50/26/bbcf64af02ae5cdde70f10fcac1b955ed6aece5de459c43feeee4d417f20/scp-0.13.0-py2.py3-none-any.whl
Requirement already up-to-date: textfsm in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: netaddr in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: pyIOSXR>=0.53 in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: cffi>=1.11.3 in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: pyeapi in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: junos-eznc>=2.1.5 in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: pynxos==0.0.3 in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Collecting setuptools>=38.4.0 (from napalm==2.3.3->-r requirements.txt (line 5))
  Downloading https://files.pythonhosted.org/packages/e7/16/da8cb8046149d50940c6110310983abb359bbb8cbc3539e6bef95c29428a/setuptools-40.6.2-py2.py3-none-any.whl (573kB)
    100% |████████████████████████████████| 573kB 2.1MB/s
Requirement already up-to-date: pyYAML in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: netmiko>=2.1.1 in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: markdown<3.0 in /usr/local/lib/python3.5/dist-packages (from py-gfm==0.1.4->-r requirements.txt (line 7))
Requirement already up-to-date: chardet<3.1.0,>=3.0.2 in /usr/local/lib/python3.5/dist-packages (from requests==2.20.1->-r requirements.txt (line 9))
Requirement already up-to-date: idna<2.8,>=2.5 in /usr/local/lib/python3.5/dist-packages (from requests==2.20.1->-r requirements.txt (line 9))
Requirement already up-to-date: certifi>=2017.4.17 in /usr/local/lib/python3.5/dist-packages (from requests==2.20.1->-r requirements.txt (line 9))
Requirement already up-to-date: urllib3<1.25,>=1.21.1 in /usr/local/lib/python3.5/dist-packages (from requests==2.20.1->-r requirements.txt (line 9))
Requirement already up-to-date: paramiko in /usr/local/lib/python3.5/dist-packages (from scp->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: lxml>=3.2.4 in /usr/local/lib/python3.5/dist-packages (from pyIOSXR>=0.53->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: pycparser in /usr/local/lib/python3.5/dist-packages (from cffi>=1.11.3->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: six in /usr/local/lib/python3.5/dist-packages (from junos-eznc>=2.1.5->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: pyserial in /usr/local/lib/python3.5/dist-packages (from junos-eznc>=2.1.5->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: ncclient>=0.5.4 in /usr/local/lib/python3.5/dist-packages (from junos-eznc>=2.1.5->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: bcrypt>=3.1.3 in /usr/local/lib/python3.5/dist-packages (from paramiko->scp->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: pyasn1>=0.1.7 in /usr/local/lib/python3.5/dist-packages (from paramiko->scp->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: pynacl>=1.0.1 in /usr/local/lib/python3.5/dist-packages (from paramiko->scp->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: cryptography>=1.5 in /usr/local/lib/python3.5/dist-packages (from paramiko->scp->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: selectors2>=2.0.1 in /usr/local/lib/python3.5/dist-packages (from ncclient>=0.5.4->junos-eznc>=2.1.5->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: asn1crypto>=0.21.0 in /usr/local/lib/python3.5/dist-packages (from cryptography>=1.5->paramiko->scp->napalm==2.3.3->-r requirements.txt (line 5))
Installing collected packages: django-tables2, scp, setuptools
  Found existing installation: django-tables2 2.0.2
    Uninstalling django-tables2-2.0.2:
      Successfully uninstalled django-tables2-2.0.2
  Found existing installation: scp 0.12.1
    Uninstalling scp-0.12.1:
      Successfully uninstalled scp-0.12.1
  Found existing installation: setuptools 40.5.0
    Uninstalling setuptools-40.5.0:
      Successfully uninstalled setuptools-40.5.0
Successfully installed django-tables2-2.0.3 scp-0.13.0 setuptools-40.6.2
Applying database migrations (python3 manage.py migrate)...
CommandError: Conflicting migrations detected; multiple leaf nodes in the migration graph: (0003_auto_20181112_0637, 0003_auto_20181112_2150 in utils).
To fix them run 'python manage.py makemigrations --merge'
Collecting static files (python3 manage.py collectstatic --no-input)...

12 static files copied to '/opt/peering-manager/static', 248 unmodified.
root@peeringmanager:/opt/peering-manager# python3 manage.py makemigrations --merge
Merging utils
  Branch 0003_auto_20181112_0637
    - Alter field action on useraction
  Branch 0003_auto_20181112_2150
    - Alter field action on useraction

Merging will only work if the operations printed above do not conflict
with each other (working on different fields or models)
Do you want to merge these migration branches? [y/N] y

Created new merge migration /opt/peering-manager/utils/migrations/0004_merge_20181115_1717.py
root@peeringmanager:/opt/peering-manager#
```

**Comment by jlixfeld** [comment]
Odd.  The two files it was complaining about are exactly the same :/

```
root@peeringmanager:/opt/peering-manager# more /opt/peering-manager/utils/migrations/0004_merge_20181115_1717.py
# Generated by Django 2.1.3 on 2018-11-15 22:17

from django.db import migrations


class Migration(migrations.Migration):

    dependencies = [
        ('utils', '0003_auto_20181112_0637'),
        ('utils', '0003_auto_20181112_2150'),
    ]

    operations = [
    ]
root@peeringmanager:/opt/peering-manager# diff ./utils/migrations/0003_auto_20181112_0637.py ./utils/migrations/0003_auto_20181112_2150.py
1c1
< # Generated by Django 2.1.3 on 2018-11-12 11:37
---
> # Generated by Django 2.1.3 on 2018-11-12 20:50
root@peeringmanager:/opt/peering-manager#
```

Re-running the upgrade.sh fixed it.

```
root@peeringmanager:/opt/peering-manager# ./scripts/upgrade.sh
Cleaning up old Python bytecode (find . -name "*.pyc" -delete)...
Updating required Python dependencies (pip3 install -r requirements.txt --upgrade)...
Requirement already up-to-date: Django==2.1.3 in /usr/local/lib/python3.5/dist-packages (from -r requirements.txt (line 1))
Requirement already up-to-date: django-filter==2.0.0 in /usr/local/lib/python3.5/dist-packages (from -r requirements.txt (line 2))
Requirement already up-to-date: django-tables2==2.0.3 in /usr/local/lib/python3.5/dist-packages (from -r requirements.txt (line 3))
Requirement already up-to-date: Jinja2==2.10 in /usr/local/lib/python3.5/dist-packages (from -r requirements.txt (line 4))
Requirement already up-to-date: napalm==2.3.3 in /usr/local/lib/python3.5/dist-packages (from -r requirements.txt (line 5))
Requirement already up-to-date: MarkupSafe==1.1.0 in /usr/local/lib/python3.5/dist-packages (from -r requirements.txt (line 6))
Requirement already up-to-date: py-gfm==0.1.4 in /usr/local/lib/python3.5/dist-packages (from -r requirements.txt (line 7))
Requirement already up-to-date: pytz==2018.7 in /usr/local/lib/python3.5/dist-packages (from -r requirements.txt (line 8))
Requirement already up-to-date: requests==2.20.1 in /usr/local/lib/python3.5/dist-packages (from -r requirements.txt (line 9))
Requirement already up-to-date: setuptools>=38.4.0 in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: cffi>=1.11.3 in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: textfsm in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: junos-eznc>=2.1.5 in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: netaddr in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: scp in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: pyYAML in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: pynxos==0.0.3 in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: pyIOSXR>=0.53 in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: pyeapi in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: netmiko>=2.1.1 in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: future in /usr/local/lib/python3.5/dist-packages (from napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: markdown<3.0 in /usr/local/lib/python3.5/dist-packages (from py-gfm==0.1.4->-r requirements.txt (line 7))
Requirement already up-to-date: urllib3<1.25,>=1.21.1 in /usr/local/lib/python3.5/dist-packages (from requests==2.20.1->-r requirements.txt (line 9))
Requirement already up-to-date: idna<2.8,>=2.5 in /usr/local/lib/python3.5/dist-packages (from requests==2.20.1->-r requirements.txt (line 9))
Requirement already up-to-date: certifi>=2017.4.17 in /usr/local/lib/python3.5/dist-packages (from requests==2.20.1->-r requirements.txt (line 9))
Requirement already up-to-date: chardet<3.1.0,>=3.0.2 in /usr/local/lib/python3.5/dist-packages (from requests==2.20.1->-r requirements.txt (line 9))
Requirement already up-to-date: pycparser in /usr/local/lib/python3.5/dist-packages (from cffi>=1.11.3->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: ncclient>=0.5.4 in /usr/local/lib/python3.5/dist-packages (from junos-eznc>=2.1.5->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: six in /usr/local/lib/python3.5/dist-packages (from junos-eznc>=2.1.5->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: pyserial in /usr/local/lib/python3.5/dist-packages (from junos-eznc>=2.1.5->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: lxml>=3.2.4 in /usr/local/lib/python3.5/dist-packages (from junos-eznc>=2.1.5->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: paramiko>=1.15.2 in /usr/local/lib/python3.5/dist-packages (from junos-eznc>=2.1.5->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: selectors2>=2.0.1 in /usr/local/lib/python3.5/dist-packages (from ncclient>=0.5.4->junos-eznc>=2.1.5->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: bcrypt>=3.1.3 in /usr/local/lib/python3.5/dist-packages (from paramiko>=1.15.2->junos-eznc>=2.1.5->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: cryptography>=1.5 in /usr/local/lib/python3.5/dist-packages (from paramiko>=1.15.2->junos-eznc>=2.1.5->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: pyasn1>=0.1.7 in /usr/local/lib/python3.5/dist-packages (from paramiko>=1.15.2->junos-eznc>=2.1.5->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: pynacl>=1.0.1 in /usr/local/lib/python3.5/dist-packages (from paramiko>=1.15.2->junos-eznc>=2.1.5->napalm==2.3.3->-r requirements.txt (line 5))
Requirement already up-to-date: asn1crypto>=0.21.0 in /usr/local/lib/python3.5/dist-packages (from cryptography>=1.5->paramiko>=1.15.2->junos-eznc>=2.1.5->napalm==2.3.3->-r requirements.txt (line 5))
Applying database migrations (python3 manage.py migrate)...
Operations to perform:
  Apply all migrations: admin, auth, contenttypes, peering, peeringdb, sessions, utils
Running migrations:
  Applying peering.0021_auto_20181113_2136... OK
  Applying utils.0003_auto_20181112_2150... OK
  Applying utils.0004_merge_20181115_1717... OK
Collecting static files (python3 manage.py collectstatic --no-input)...

0 static files copied to '/opt/peering-manager/static', 260 unmodified.
root@peeringmanager:/opt/peering-manager#
```

**Comment by jlixfeld** [comment]
I don't see much of a difference after this commit.  My resolution varies depending on where I am.  Home it's 3360x1890, work it's 2560x1440, remotely it's 2880x1800

**Comment by respawner** [comment]
I see, I have a 2560x1440 at home and it looks OK (at least whit Chrome) but on bigger resolutions like yours I understand the issue. Should we make the main container width 4K ready?

**Comment by jlixfeld** [comment]
Ah, interesting.  It works for me too in Chrome, but not in Safari (my normal browser).

I suppose it makes sense to make it 4K ready, doesn't it?

**Comment by respawner** [comment]
Should be addressed in #81.
